### ISSUE TITLE
Accessibility -  live region in Xamarin.Forms ( WAI-ARIA in html)

### ISSUE THREAD

**Issue by PureWeen** [dotnet/maui#3997]
### Description

Imagine that we have a zone or several zones of our App whose content is added, deleted, updated or modified without user intervention.

It is necessary to tell the screen reader what is happening.

Problem, the solutions in IOS is very different from:

Android:
https://codelabs.developers.google.com/codelabs/basic-android-accessibility/#6

UWP:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationlivesetting

IOS:
https://developer.apple.com/documentation/foundation/nsnotificationcenter/1410472-postnotification?language=objc

### Public API Changes

TBD

### Intended Use-Case

Steps to reproduce.

1) Enable the screen reader.
2.0)  Make a "Pull To Refresh", the screen reader does not receive any indication
2.1) Click on a button that makes a call to a service. (If the service takes 30 seconds) the screen reader receives no indication.
2.2) other cases

Provisional solution applied.

Create a service and verify if the screen reader is activated, if this is the case, send a Toast to the user to notify the screen reader. In the case of IOS, we directly use the PostNotification method

------------------------------------------------

In this demo, I try to simulate one of the problems on button 1, by not notifying the user what is happening on the screen. (when an App is busy and the screen reader does not receive any notification)
https://github.com/elbrinner/accesibilidad-apps-xamarin-forms
